package.json

```diff
@@ -46,6 +46,8 @@
     "d3-time-format": "^2.0.0",
     "element-resize-detector": "^1.1.6",
     "lodash": "^4.11.0",
+    "moment": "^2.17.1",
```

Comment:
With path imports at least, only people who are using `DateSelect` will be impacted, and it's likely they will be using `moment` in their applications in that case.

Still agree, though, if we can avoid adding a dependency at all that's always better.